model/registry/azure.yaml

```diff
@@ -0,0 +1,67 @@
+groups:
+  - id: registry.azure
+    prefix: azure
```

Comment:
The PR also defines `az.service_request_id`.

Reply:
It's defined here: 

https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/32486/files#diff-fbffe677c472f0c2f632302b809514af9861cd93c2490025a1be3d80b76a7e72R23

Reply:
Also, the prefix on that is `az` not `azure`